config/gunicorn/dev.py

```diff
@@ -7,7 +7,7 @@
 # The number of worker processes for handling requests
 workers = 2
 # The socket to bind
-bind = "0.0.0.0:8000"
+bind = "0.0.0.0:127"
```

Comment:
Is this a typo, or are we wanting to bind gunicorn to port 127?

Reply:
This is what I thought you meant when you said it should be on 127 for security reasons. Unless you meant change 0.0.0.0 to 127.0.0.1?

Reply:
Yes, I meant it should be listening specifically on the loopback address - 127.0.0.1